docs/commands/tmux-setup.md

```diff
@@ -13,10 +13,18 @@ configure the context to be redirected to the new pane, looking something like:
 ![](https://i.imgur.com/Khk3xGl.png)
 
 To set it up, simply enter
+
 ```
 gef➤ tmux-setup
 ```
 
 **Note**: Although `screen-setup` provides a similar setup, the structure of
 `screen` does not allow a very clean way to do this. Therefore, if possible, it
 would be recommended to use the `tmux-setup` command instead.
+
+### Possible color issues with tmux ###
+
+On Linux tmux only supports 8 colors with some terminal capabilities (`$TERM`
```

Comment:
I have never had this issue. You can/should just set `TERM=screen-256color`

Reply:
That is one solution but I, for example, sometimes use other `TERM`s that don't work well with tmux and therefore when opening tmux it falls back to e.g. `screen` and then I get these problems and it took me a while to figure this out :) (Of course I am aware that I could just override my regular TERM with `~/.tmux.conf`. But first one needs to be aware of the underlying issue).

But if you want me to remove this section because it's not related to GEF at all, I completely understand.

Reply:
It's fine, I just want to make sure we give a good suggestion. Let's add both suggestions, you can explain why you use -2